packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -5,22 +5,23 @@ import {BigNumber, constants, ethers, providers} from 'ethers';
 import {fromEvent} from 'rxjs';
 import {take} from 'rxjs/operators';
 
+import {ChainService} from '../chain-service';
 import {defaultTestConfig} from '../config';
 import {ObjectiveSucceededValue, SingleChannelOutput, Wallet} from '../wallet';
 import {getChannelResultFor, getPayloadFor} from '../__test__/test-helpers';
 
-if (!defaultTestConfig.rpcEndpoint) throw new Error('rpc endpoint must be defined');
-const rpcEndpoint = defaultTestConfig.rpcEndpoint;
 let provider: providers.JsonRpcProvider;
-const b = Wallet.create({
+
+const GANACHE_RPC_ENDPOINT = 'http://localhost:8545';
```

Comment:
This seems like a bad proliferation of a magic constant.

Reply:
Right now on `master` the jest tests in the `server-wallet` that use a chain all hard code the usage of `http://localhost:8545` inside `chain-setup.ts`. I think this change just makes it much more obvious. I think we may want to open a separate issue if we want to refactor the tests to easily have configurable chain endpoints.